app/views/users/dossiers/identite.html.haml

```diff
@@ -13,7 +13,7 @@
       champs requis
 
     = f.label :gender, class: "required"
-    = f.select :gender, [Individual::GENDER_MALE, Individual::GENDER_FEMALE], {}, class: "small"
+    = f.select :gender, [Individual::GENDER_MALE, Individual::GENDER_FEMALE], { :selected => Individual::GENDER_MALE }, class: "small"
```

Comment:
Au sujet de 

>  Make the 'male' field selected by default during identity creation
>
> This is necessary for screen readers, but very unfortunate (from a gender-neutral point of view). I see no good solution:
> - having no selected value is bad for users, it will force many to click once or twice, which can be skipped here
> - making Female selected by default movs the problem

En plus des soucis d'égalité de genre, ça m'embête un peu d'avoir une option sélectionnée par défaut qui n'est pas forcément la bonne. J'ai l'impression qu'ailleurs dans l'app on évite de faire ça (par exemple en ne sélectionnant pas un "Oui" ou "Non" par défaut, ni une option dans un menu déroulant), parce qu'on court le risque que l'usager ignore le champ, et ne renseigne pas explicitement une valeur.

Est-ce qu'on n'aurait pas moyen de mettre une valeur par défaut juste pour les lecteurs d'écran, ou qqc comme ça ?

Reply:
En fait, c'est déja 'Mr.' qui est sélectionné par défaut actuellement (au sens où c'est la valeur du champ, mais il n'a pas l'attribut `selected`).
J'ai l'impression qu'on peut soit:
 - ajouter une valeur vide pour forcer les usagers à choisir, et donc il n'y aurait plus de problème de valeur sélectionnée
 - garder le comportement actuel, et fournir l'info aux lecteurs d'écrans avec [`aria-selected`](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-selected).

Reply:
Oooh, d'accord, compris !

Alors je dirais :

- Idéalement, on transforme ce `select` en `radio` (parce qu'un `select` à seulement 2 valeurs c'est absurde), et ça le rendra accessible et tout.
- Mais sinon, au plus simple, on met une valeur vide pour forcer les gens à choisir.

Reply:
J'ai opté pour le bouton radio (sans `.inline-champ`, qui structure mieux mais rend assez bizarre). Il reste des labels orphelins, mais je me rends compte qu'il y a des trucs dans les autres PR à merger pour récupérer un bout du travail à ce sujet.
![image](https://user-images.githubusercontent.com/1223316/77433070-2f7bd280-6ddf-11ea-8588-5cdb65a1228c.png)